pkg/controller/account/account_controller.go

```diff
@@ -520,6 +531,25 @@ func CreateAccount(reqLogger logr.Logger, client awsclient.Client, accountName,
 	return accountStatus, nil
 }
 
+func checkAWSAccountsLimitReached(r *ReconcileAccount, reqLogger logr.Logger, currentAccounts int) (bool, error) {
```

Comment:
Is this function related to this PR?

Reply:
No, it's not. Not sure how that got introduced.  I'll fix it.